src/components/general/button/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactElement } from 'react';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { ContextualColors } from '../../../styles';
+import { Icon } from '../../../types/icon';
+import { registerIcons } from '../../../registerIcons';
+registerIcons();
```

Comment:
this is not specific to the button, you should do this somewhere else closer to the root of the project